### PR TITLE
Milestone 5: Event handlers

### PR DESCRIPTION
**Pull request by Ceci007**
## Milestone 5: Event handlers
In this project I:
- Set state in `App.js`.
- Pass event handlers as props from Parent to Children components.

### CODE REVIEW

File: src/components/ButtonPannel.js [hunk 1]
```diff
@@ -1,61 +1,46 @@
...
     </div>
   );
+}
+
+ButtonPanel.propTypes = {
+  onClick: PropTypes.func.isRequired,
 };
 
-export default ButtonPannel;
+export default { ButtonPanel };
```
Comment (christianotieno) [inline src/components/ButtonPannel.js:46]: ## prop-type error
- [ ] Our button props are going in as undefined 😟 
prop-error

File: src/logic/calculate.js [hunk 2]
```diff
@@ -1,55 +1,69 @@
...
+    case '÷':
+      if (newCalc.total == null) {
+        newCalc.total = newCalc.next;
+      } else {
+        newCalc.total = operate(newCalc.next, newCalc.total, newCalc.operation);
+      }
+      newCalc.next = null;
+      newCalc.operation = buttonName;
+      break;
+    case '%':
```
Comment (christianotieno) [inline src/logic/calculate.js:42]: ## Improve the calculate.js functionality
- [ ] The first value defaults to `0` though it should be displaying the value the user typed in. React is instead ignoring it and replacing it with `0` like below, I pressed `8` in the previous input. 👇 
zero-default

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @Ceci007 👋 

Most of the code is meeting the majority of the requirements in this milestone. We, however, do have some issues that need addressing as I have highlighted below. 👇 

The proposed changes touch on the previous value displayed as the default value `0` after a user has pressed the operator `button` instead of the value itself and some `buttonPanel`s props errors in the console. 
### Happy coding 💻 
#### Christian Otieno (CR)

**Review by Torres-ssf — APPROVED** [review]
### Milestone approved :+1: 
Hi @Ceci007 :hand: 
Awesome job implementing the requested changes.
You may merge the PR

Happy coding :computer: 
**Sergio Torres**

**Pull request merged**